requirements/load_test.txt

```diff
@@ -0,0 +1,7 @@
+# Load Testing Requirements
```

Comment:
do we need to update any other docs about the requirements installation?

Reply:
I don't think so, as it's entirely specific to the load testing and inherits from the base.